dialog.mdl

```diff
@@ -5372,7 +5372,9 @@ dialogs =
         (TeamPaneImage, SideRight, Point(10, -150), Point(-750, -150), .5, ofTeam | ofExpandedTeam, false),
 
         // MergedInvestmentsPaneImage: the investments pane (F5)
-        (MergedInvestmentsPaneImage, SideCenter, Point(-192, 2000), Point(-192, -265), .5, ofInvestment, false),
+        //Change from Bunnywabbit to make investor pane placement make more sense

```

Comment:
@Xynth22 I'm in favor of no longer having historical information stored in the code. That's what git is great for.

Let's remove lines 5375 and 5376.